Frameworks/Foundation/NSFileHandle.mm

```diff
@@ -175,37 +121,103 @@ + (instancetype)fileHandleForUpdatingAtPath:(NSString*)file {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
```

Comment:
The header file must be updated too, to remove the STUB_* macros (and add NOTINPLAN_* macros where applicable.